content/docs/guides/ads_on_amp/ads_getting_started.md

```diff
@@ -0,0 +1,63 @@
+---
+$title : "Getting Started"
+$order : 0
+$parent: /content/docs/guides/ads_on_amp
```

Comment:
Huh, it ran for me and I'm able to preview it even now. But it looks like I may not even need this $parent line; I'll try getting rid of it.